docs/reference/filebeat/filebeat-input-journald.md

```diff
@@ -8,6 +8,18 @@ mapped_pages:
 
 [`journald`](https://www.freedesktop.org/software/systemd/man/systemd-journald.service.html) is a system service that collects and stores logging data. The `journald` input reads this log data and the metadata associated with it. To read this log data Filebeat calls `journalctl` to read from the journal, therefore Filebeat needs permission to execute `journalctl`.
 
+:::{important}
+When using the Journald input from a Docker container, make sure the
+`journalctl` binary in the container is compatible with your
+Systemd/journal version, to get the version of the `journalctl` binary
+in Filebeat's image run (adjust the image name/tag accordingly to the
+version you are running):
```

Comment:
```suggestion
When using the Journald input from a Docker container, make sure the
`journalctl` binary in the container is compatible with your
Systemd/journal version. To get the version of the `journalctl` binary
in Filebeat's image run the following, adjusting the image name/tag 
according to the version that you are running):
```

Reply:
Thanks @kilfoyle ! I implemented it on 02f0929502